app/models/dialog.rb

```diff
@@ -1,8 +1,9 @@
 class Dialog < ApplicationRecord
-  DIALOG_DIR = Rails.root.join("product/dialogs/service_dialogs")
+  DIALOG_DIR_CORE   = 'product/dialogs/service_dialogs'.freeze
+  DIALOG_DIR_PLUGIN = 'content/service_dialogs'.freeze
```

Comment:
you mean move all from `product/dialogs/service_dialogs` to `content/service_dialogs`?